integrations/docker/images/chip-build-efr32/Dockerfile

```diff
@@ -29,9 +30,10 @@ RUN wget https://www.silabs.com/documents/login/software/slc_cli_linux.zip && \
 
 ENV PATH="${PATH}:/slc_cli/"
 
-#  TODO Fix me
-# RUN slc configuration  --sdk="$GSDK_ROOT" && \
-#     slc signature trust --sdk "$GSDK_ROOT"
-
+# Install Python Packages
```

Comment:
Based on logs now I see that the above command on line 29 will fail since requirements.txt seems gone from the zip file.